src/test/integration/patchNamespace.ts

```diff
@@ -0,0 +1,47 @@
+import assert, { deepEqual } from 'node:assert';
+import { PatchStrategy, CoreV1Api, KubeConfig, setHeaderOptions, V1Namespace } from '../../index.js';
+
+export default async function patchNamespace() {
+    const kc = new KubeConfig();
+    kc.loadFromDefault();
+
+    const coreV1Client = kc.makeApiClient(CoreV1Api);
+    let newNS = new V1Namespace();
+    const testNSName = 'integration-test-patch-ns';
```

Comment:
If we would add a variable at the end i.e. some random generated string or use `generateName` instead of name we could run the test multiple times against the same cluster without any cleanup work.

We may don't need that often, but it could be good for local debugging.

Reply:
agreed